Dockerfile

```diff
@@ -17,5 +17,5 @@ RUN pip install -r requirements.txt
 EXPOSE 8008
 EXPOSE 8006
```

Comment:
redundant?

Reply:
Not if developer wants to setup only the metax-web container